src/generator.js

```diff
@@ -221,7 +221,7 @@ import TabItem from '@theme/TabItem';
    */
   mdxLinks(text) {
     for (const name of this.generatedFiles)
-      text = text.replace(new RegExp('\\' + name, 'g'), name + 'x');
+      text = text.replace(new RegExp(`/(${path.basename(name)})`, 'g'), '/$1x');
```

Comment:
`name = './api/class-testinfo.md'`

before it was matching for absolute paths, now it just matches for `/foo-bar.md` -> `/foo-bar.mdx` because it starts either with `..` or `.`.